keycloak/keycloak-admin-ui
# Realm roles: associated roles tab

## Motivation


#285 , #286, https://issues.redhat.com/browse/APPDUX-728

## Brief Description


Implements attributes tab to resemble the mock: https://marvelapp.com/prototype/f39d464/screen/72684631

This PR is meant to address the layout of the attributes tab and display the correct composite realm roles once they are added by a user. 

There is a separate issue in JIRA to address removing realm roles (https://issues.redhat.com/browse/APPDUX-747) and this work will be continued in a separate PR.

## Verification Steps


1. Go to Realm Roles section. 
2. Create a new role
3. Click on the "Action" menu in the upper left
4. Select "Add associated roles"
5. Add associated roles
6. Verify the "Associated roles tab" is selected
7. Go to the "action" menu again
8. Select "delete all associated roles"
9. Verify the functionality works as expected 
10. Click "Add role" button within the "associated roles tab"
11. Add an associated role
12. Verify the functionality works as expected


## Checklist:


## Additional Notes
